hotshot-query-service/src/fetching/provider/query_service.rs

```diff
@@ -511,6 +511,9 @@ mod test {
         future::{join, FutureExt},
         stream::StreamExt,
     };
+    // generic-array 0.14.x is deprecated, but VidCommitment requires this version
+    // for From<Output<H>> impl in jf-merkle-tree
+    #[allow(deprecated)]
```

Comment:
@mrain do you have a better idea on how to fix this, so we don't need to use deprecated generic-array? It seems the latest `digest` still uses the deprecated version.